app.model.js

```diff
@@ -15,4 +15,13 @@ return fs.readFile("./endpoints.json", "utf-8").then((data) => {
   }) 
   }
 
-module.exports = {getAllTopicsInfo, getAllEndpointsInfo};
+  function getArticleInfo(article_id) {
+    return db.query("SELECT * FROM articles WHERE article_id = $1", [article_id]).then((article) => {
+      if (article.rows.length === 0) {
+        return Promise.reject({status : 404, msg : 'article id does not exist'})
+      }
+      return article.rows[0]
+    });
+  }
```

Comment:
Great model here - your custom error handling and parameterized query are on point